test/api-test/api/v3/search/jobcodes/get.json

```diff
@@ -14,6 +9,11 @@
     "targetNodeName": "Computer Systems Analysts",
     "frameworkName": "o*net"
   },
+  {
```

Comment:
It looks like you changed the result sort order in the assertions? Is this intentional? Please advise?